### PR TITLE
GC controller support for desktops

### PR DESCRIPTION
**Pull request by BernardZhao**
Need to test on desktops, but I wanted some help since I'm not too familiar with Puppet.

### CODE REVIEW

File: modules/ocf/manifests/packages/dolphin_emu.pp [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class ocf::packages::dolphin_emu {
```
Comment (dkess) [inline modules/ocf/manifests/packages/dolphin_emu.pp]: If this package doesn't have a special setup needed to install it, you don't need to make a new Puppet class for it, just add it to the list of desktop packages at https://github.com/ocf/puppet/blob/master/modules/ocf_desktop/manifests/packages.pp

**Comment by cg505** [comment]
isn't dolphin in debian super old or something?

**Comment by BernardZhao** [comment]
> 
> 
> isn't dolphin in debian super old or something?

It definitely is, but honestly I'm not sure how to check what the version in debian even is,

**Comment by kpengboy** [comment]
@BernardZhao packages.debian.org

**Comment by BernardZhao** [comment]
After testing the version in Debian and a version I built myself, the performance is just too poor to justify installing the official version. I guess if people want to play, we can just play off of my nfs.

**Comment by Baisang** [comment]
@BernardZhao Please squash your commits first

**Pull request merged**